spring-data-relational/src/main/java/org/springframework/data/relational/core/mapping/RelationalPersistentEntity.java

```diff
@@ -52,7 +52,9 @@ default SqlIdentifier getQualifiedTableName() {
 	 * Returns the column representing the identifier.
 	 *
 	 * @return will never be {@literal null}.
+	 * @deprecated use {@code AggregatePath.getTableInfo().getIdColumnInfos()} instead.
```

Comment:
We should also mention that an entity can have multiple identifier columns.

Reply:
Done